### ISSUE TITLE
Unauthorized

### ISSUE THREAD

**Issue by Wh1teMag1c** [borisenko09/DMarketAPI#1]
When using some requests, it gives the request status: 401 and the following message "{'Code': 'Unauthorized', 'Message': 'Unauthorized'}", I'm doing everything exactly as it says in the api, but I can't fix it, if you know what the problem is, please write

**Comment by lreflectl** [comment]
Yeah, seems like they have updated authentication method for the api. Currently method described in their api docs doesn't work.

**Comment by borisenko09** [comment]
> When using some requests, it gives the request status: 401 and the following message "{'Code': 'Unauthorized', 'Message': 'Unauthorized'}", I'm doing everything exactly as it says in the api, but I can't fix it, if you know what the problem is, please write

I collected the basic API requests that the documentation provides, checked those that were interesting to me, they were working, write what request you tried
